src/components/PathNav.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import React from "react";
+import { usePathname } from "next/navigation";
+
+interface PathNavParams {
+  pathInfo: PathInfoType[];
+}
+
+export type PathInfoType = {
+  display: string;
+  url: string;
+};
+
+const PathNav = ({ pathInfo }: PathNavParams) => {
+  const currentPath = "http://localhost:3000" + usePathname();
+  const totalPathLength = currentPath.length;
+
+  const getRemovePath = (currentIndex: number) => {
```

Comment:
I think this function is a bit hard to read. My understanding is `currentIndex` represents the segment of the url you're looking for. To remove everything above the index, you want to remove the subtring by the length of all segments from `currentIndex + 1` to end of array? Can you guys confirm @selenaz03 @nathan-j-edwards?

I think we can opt-in for a more readable approach using list operations.

```javascript
const getPath = (currentIndex: number) => {
  const segments = currentPath.split("/");
  const path = segments.slice(0, segments.length - (pathInfo.length - currentIndex - 1))
  return path.join("/")
}
```